log-collector-script/windows/README.md

```diff
@@ -0,0 +1,114 @@
+### <span style="font-family: times, serif; font-size:16pt; font-style:italic;"> EKS Logs Collector 
+
+<span style="font-family: calibri, Garamond, 'Comic Sans MS' ;">This project was created to collect Amazon EKS log files and OS logs for troubleshooting Amazon EKS customer support cases.</span>
+
+#### Usage
+* Collect EKS logs using SSM agent, jump to below [section](#collect-eks-logs-using-ssm-agent) _(or)_
+
+* Run this project as the Administrator user:
+```
+Invoke-WebRequest -OutFile eks-log-collector.ps1 https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/log-collector-script/windows/eks-log-collector.ps1
+.\eks-log-collector.ps1
+```
+
+#### Example output
+The project can be used in normal or Enable/Disable Debug(**Caution: Enable/Disable Debug will restart Docker daemon which would kill running containers**).
+
+```
+# sudo bash eks-log-collector.sh --help
+USAGE: eks-log-collector --help [ --mode=collect|enable_debug --ignore_introspection=true|false --ignore_metrics=true|false ]
```

Comment:
I don't `ignore_introspection`, `ignore_metrics` functions in the script. Can you update the readme section to map help from the script.

Reply:
I have updated the readme for the same.